components/scream/src/control/tests/ad_tests.yaml

```diff
@@ -4,20 +4,21 @@ Debug:
   Atmosphere DAG Verbosity Level: 5
 
 Initial Conditions:
-  Initial Conditions File: should_not_be_neeeded_and_code_will_throw_if_it_tries_to_open_this_nonexistent_file.nc
-  A: 1.0
-  B: 2.0
-  C: 3.0
-  D: [4.0, 5.0]
-  E: "A"
+  Point Grid:
+    Filename: should_not_be_neeeded_and_code_will_throw_if_it_tries_to_open_this_nonexistent_file.nc
+    A: 1.0
+    B: 2.0
+    C: 3.0
```

Comment:
Not relevant to this PR, but I'm mentioning now because it just reminded me.  The YAML parser interprets any number starting with `-` as a string not a double.  So it's impossible to assign a negative value in this way.  We may want to fix that at some point.

Reply:
Do you mean that if you tell the parser to interpret negative numbers as doubles, that operation fails? The parser can interpret anything as a string, and you tell it to cast to a specific data type, right?

Reply:
Or are we using something on top of yaml-cpp to parse our YAML files?

Reply:
Really?!? Wow, I didn't know that. There should be a way to fix this.

Reply:
Interesting. In ekat, the unit test for yaml parser correctly reads stuff like `My Int: -2` as a negative int. But `-2.0` is read as a string. I guess I should fix this right away in ekat.